playground/tools/scrap.py

```
@@ -0,0 +1,357 @@
+import requests
+from typing import List, Dict, Any
+from langchain.tools import tool
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+session = requests.Session()
+session.headers.update({
+    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36",
```

Ruff (0.12.2)

Line too long (138 > 88)

(E501)